packages/design-system/src/molecules/accountInfoDropdown/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { FlatList, Pressable, StyleSheet, View } from 'react-native'
+import { CircleImg, Icon, IconVariant, TypographyV2 } from '../../atoms'
+import { useThemeV2 } from '../../hooks'
+import { Dropdown } from '../dropdown'
+import { container, margin, padding } from '../../styles'
+
+const AVATAR_SIZE = 40
+const DROPDOWN_WIDTH = 282
+
+export interface AccountInfoDropdownProps {
+  avatar: string
+  name: string
+  username: string
+  children: React.ReactNode
+  options: {
+    id: string
+    iconVariant: IconVariant
+    color?: string
+    label: string
+    onPress: () => void
+  }[]
+}
+
+const styles = StyleSheet.create({
+  container: {
+    width: DROPDOWN_WIDTH,
+  },
+  divider: {
+    height: 1,
+  },
+})
+
+const renderListSeparator = () => (
+  <View style={[margin.mv8, container.fullWidth]} />
+)
```

Comment:
We can make it a static value so we don't create a new component each time the function is called.

```suggestion
const listSeparator = <View style={[margin.mv8, container.fullWidth]} />
```

Reply:
Element is not an allow value type for ItemSeparatorComponent prop
```
Type 'Element' is not assignable to type 'ComponentType<any> | null | undefined'.